build/Publish.targets

```diff
@@ -288,4 +296,31 @@
                     MaxClients="$(PushToBlobFeed_MaxClients)" />
   </Target>
 
+  <Target Name="_PublishDeb">
+    <Error Text="PublishDebianInstallerServer must be set as a MsBuild Property variable for debian publishing." Condition="'$(PublishDebianInstallerServer)' == ''" />
+    <Error Text="PublishDebianInstallerRepoIDs must be set as a MsBuild Property variable for debian publishing." Condition="'$(PublishDebianInstallerRepoIDs)' == ''" />
+    <Error Text="PublishDebianInstallerUser must be set as a MsBuild Property variable for debian publishing." Condition="'$(PublishDebianInstallerUser)' == ''" />
+    <Error Text="PublishDebianInstallerPassword must be set as a MsBuild Property variable for debian publishing." Condition="'$(PublishDebianInstallerPassword)' == ''" />
+
+    <ItemGroup>
+      <RepoIds Include="$(PublishDebianInstallerRepoIDs)" />
+    </ItemGroup>
+
+    <UploadToLinuxPackageRepository
+      Server="$(PublishDebianInstallerServer)"
+      RepositoryId="%(RepoIds.Identity)"
+      Username="$(PublishDebianInstallerUser)"
+      Password="$(PublishDebianInstallerPassword)"
+      PathOfPackageToUpload="$(DebFile)"
+      PackageNameInLinuxPackageRepository="$(DebName)"
+      PackageVersionInLinuxPackageRepository="$(DebVersion)" />
+  </Target>
+
+  <Target Name="PublishDebs" DependsOnTargets="GetFilesToPublish">
```

Comment:
Add a condition here: `Condition="'$(PublishToDebianFeed)' == 'true'"` and use `$(PublishType.Contains('debian'))` to determine when this should happen. e.g. this would be skipped for prodcon builds.

https://github.com/aspnet/Universe/blob/5d670bf6b1b5fbe06942d00b0e27e1709396ee1e/build/repo.props#L47-L56